app/jobs/risc_delivery_job.rb

```diff
@@ -64,6 +71,13 @@ def perform(
         error: err.message,
       }.to_json,
     )
+
+    track_event(
```

Comment:
[question] for the exceptions that are leading to retries, i put the event here so it would only trigger if it ultimately failed. do we want to move it up so there is an event for every attempt?